### ISSUE TITLE
i can not disable certification verification

### ISSUE THREAD

**Issue by vampywiz17** [fedarovich/qbittorrent-cli#5]
I try to disable cert. verify, with this command: ` qbt network settings -i true`

But nothing happened:
```
qbt network settings

Use default credentials:   False
Ignore certificate errors: False

```

It is a Windows 10 system, a latest version of qbittorrent-cli.

**Comment by fedarovich** [comment]
Thank you for reporting the issue. I'll check it as quickly as I can.

**Comment by vampywiz17** [comment]
Working, thanks!

**Comment by fedarovich** [comment]
Fixed in version 1.2.18320.1
